realm/realm-annotations-processor/src/main/java/io/realm/processor/ClassMetaData.kt

```diff
@@ -698,7 +698,9 @@ class ClassMetaData(env: ProcessingEnvironment, typeMirrors: TypeMirrors, privat
     }
 
     // The field has the @PrimaryKey annotation. It is only valid for
-    // String, short, int, long and must only be present one time
+    // String, short, int, long and must only be present one time.
+    // From Core 6 primary keys are not indexed pr default anymore as
```

Comment:
```suggestion
    // From Core 6 primary keys are not indexed by default anymore as
```

Reply:
Actually in Core 6 String primary keys don’t need an Index, and in Core 10 none of the primary keys do